### DISCUSSION TITLE
SE_HW3_14022_Q2

### DISCUSSION THREAD

**Discussion by AmooHashem** [ssc-public/Software-Engineering#8] [Q&A]
سوالات خود پیرامون سوال دوم از تمرین سوم درس در ترم ۱۴۰۲۲ را اینجا بپرسید!

**Comment by mikakzm** [comment]
> سوالات خود پیرامون سوال دوم از تمرین سوم درس در ترم ۱۴۰۲۲ را اینجا بپرسید!

فقط معیار های کیفیتی که مورد تاثیر قرار میگیرند رو کافیه برای قسمت سوم نام ببریم یا توضیح اضافه ای لازم هست؟

**Reply by armindelgosar** [reply]
علاوه بر نام بردن، توضیح مختصری درباره هر کدام از معیار ها بیان کنین که مشخص بشه با استناد به چه دلایلی این معیارها تعریف شده. ولی توضیح بیش از حد لازم نیست.

**Reply by mehrafarin-k** [reply]
> علاوه بر نام بردن، توضیح مختصری درباره هر کدام از معیار ها بیان کنین که مشخص بشه با استناد به چه دلایلی این معیارها تعریف شده. ولی توضیح بیش از حد لازم نیست.

متوجه نمیشم که یعنی چی با استناد به چه دلایلی معیار ها تعریف شدن. یعنی بگیم متریکی که باهاش فهمیدن این معیار کیفی مورد تاثیر قرار گرفته چی بوده؟

**Reply by armindelgosar** [reply]
بله توضیح بدید که این معیار هایی که برای کیفیت نرم افزار هستن، فارغ از بحث الگو های طراحی از کجا اومدن و چرا مهم هستن. بعد اشاره کنید وجود الگو های طراحی در حوزه نرم افزار و استفاده از انها، همون معیار هارو به چه صورت روش اثر میزاره. تحلیل کنین که روی هر معیار مثلا فلان تاثیر رو گذاشته.
